platform/ios/src/Mapbox.template.h

```diff
@@ -77,6 +77,6 @@ FOUNDATION_EXPORT MLN_EXPORT const unsigned char MapboxVersionString[];
 #import "NSPredicate+MLNAdditions.h"
 #import "NSValue+MLNAdditions.h"
 #import "MLNUserLocationAnnotationViewStyle.h"
-#if MLN_DRAWABLE_RENDERER
+#if defined(MLN_DRAWABLE_RENDERER) || defined(MLN_RENDER_BACKEND_METAL)
```

Comment:
@louwers : The Custom Drawable Layer works both for GL and Metal flavors of Drawable. The incompatibility is with Legacy only.